pkg/controller/controller_instance.go

```diff
@@ -1168,18 +1168,17 @@ func (c *controller) recordStartOfServiceInstanceOperation(toUpdate *v1beta1.Ser
 	toUpdate.Status.CurrentOperation = operation
 	now := metav1.Now()
 	toUpdate.Status.OperationStartTime = &now
+	toUpdate.Status.InProgressProperties = inProgressProperties
```

Comment:
Set `InProgressProperties` for all operations, including `Deprovision`. It will guarantee that we are passing correct plan ref.